.github/workflows/test.yml

```diff
@@ -72,3 +72,49 @@ jobs:
       with:
         name: cadvisor.log
         path: ${{ github.workspace }}/go/src/github.com/google/cadvisor/cadvisor.log
+  test-integration-s390x:
```

Comment:
It would be great if you could upload logs on error - they can help a lot when trying to understand why tests are failing, especially on architecture that some of us might not be able to get their hands on.

Reply:
Test cases are failing in Github Actions as docker is not coming up in s390x container under qemu. They pass in s390x container on s390x VM. After this change 
```
printf '{"iptables": false,"ip-masq": false,"bridge": "none" }' > /etc/docker/daemon.json
```
I could get docker up in s390x container under qemu but it fails to pull images, throws this error
```
Attempting next endpoint for pull after error: failed to register layer: Error processing tar file(exit status 1): 
```
cc @dims

Reply:
According to the [documentation](https://docs.docker.com/engine/reference/commandline/dockerd/):

> `--iptables=false` prevents the Docker daemon from adding iptables rules. If multiple daemons manage iptables rules, they may overwrite rules set by another daemon. Be aware that disabling this option requires you to manually add iptables rules to expose container ports. If you prevent Docker from adding iptables rules, Docker will also not add IP masquerading rules, even if you set `--ip-masq` to `true`. Without IP masquerading rules, Docker containers will not be able to connect to external hosts or the internet when using network other than default bridge.

Reply:
`uraimo/run-on-arch-action` will execute your commands (`run` on line 91) docker will  not be able to pull any images that are needed to run integration tests.